### PR TITLE
Security Checklist

### PR DESCRIPTION
**Pull request by lezzokafka**
Adding Validator Security Checklist.

### CODE REVIEW

File: docs/getting-started/security-checklist.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+| Data Center       | Redundant Cooling               |
+| Data Center       | Redundant Networking            |
+| Data Center       | Physical Cage/Gated Access      |
+| Data Center       | Remote Alerting Security Camera |
+
+
+
+## Part 2 - Current Status of Node Setup
+
+Description: Perform a survey on your current status of node setup , and compare your result with the best practice belo
```
Comment (lezzokafka) [inline docs/getting-started/security-checklist.md:21]:
```suggestion
Description: Perform a survey on your current status of node setup, and compare your result with the best practice suggested below
```

File: docs/getting-started/security-checklist.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Crypto.com Mainnet Validator Security Checklist
+
+## Part 1 - Conduct Survey on General Controls of Hosting Data Centre
+
+Description: Perform a survey on the hosting data centre, and compare your result with the best practice below
```
Comment (lezzokafka) [inline docs/getting-started/security-checklist.md:5]:
```suggestion
Description: Perform a survey on the hosting data centre, and compare your result with the best practice suggested below
```

**Review by CeruleanAtMonaco — APPROVED** [review]
ok la for now

File: docs/getting-started/security-checklist.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+| Key Managment                           | Setup Tendermint KMS with HSM or equivalent online service, which should replace the static key file.                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| DDOS                                    | Setup validator in accordance with sentry architecture. Setup [instruction](https://docs.tendermint.com/master/nodes/validators.html#setting-up-a-validator) and [detailed description](https://forum.cosmos.network/t/sentry-node-architecture-overview/454).                                                                                                                                                                                                                                                                          |
```
Comment (lezzokafka) [inline docs/getting-started/security-checklist.md:43]:
```suggestion
| DDOS                                    | Setup validator in accordance with sentry architecture. Kindly refer to the setup [instruction](https://docs.tendermint.com/master/nodes/validators.html#setting-up-a-validator) and [detailed description](https://forum.cosmos.network/t/sentry-node-architecture-overview/454).                                                                                                                                                                                                                                                                          |
```

**Pull request merged**